netci.groovy

```diff
@@ -31,4 +32,33 @@ def branch = GithubBranchName
     }
 }
 
+
+// Generate a fake job to test ReproBuild functionality
+def reproJob = job(Utilities.getFullJobName(project, 'Windows_NT_ReproBuild', true)) {
+    steps {
+        def curlCommand = """Invoke-RestMethod https://snapshotter.azurewebsites.net/api/snapshot/vm?code=%SNAPSHOT_TOKEN% -Method Post -Body "{ 'group': 'dotnet-ci1-vms', 'name': '%COMPUTERNAME%', 'targetGroup': 'snapshot-test' }" -ContentType 'application/json'"""
```

Comment:
I think in this case `%SNAPSHOT_TOKEN%` needs to be `$env:SNAPSHOT_TOKEN` because this is running in powershell now.